### PR TITLE
light mode desktop landing page

### PR DESCRIPTION
**Pull request by towardsnewtech**

### CODE REVIEW

File: components/LandingPage/LandingPage.tsx [hunk 1]
```diff
@@ -1,18 +1,30 @@
...
-  <div className="h-screen overflow-y-auto text-black bg-white">
-    <SeoHead title="Cre8ors" description="Cre8ors" image="/CRE8ORSLOGO_ICON.svg" />
-    <Header />
-    <Hero />
-    <Content />
-    <Community />
-  </div>
-)
+const LandingPage = () => {
+    return (
```
Comment (sweetmantech) [inline components/LandingPage/LandingPage.tsx]: nit
you don't need the return statement here

instead you could use:
```
const LandingPage = () => (
...componentRender
)
```

File: components/LandingPage/LandingPage.tsx [hunk 2]
```diff
@@ -1,18 +1,30 @@
...
+            <SeoHead title="Cre8ors" description="Cre8ors" image="/CRE8ORSLOGO_ICON.svg" />
+            <Header />
+            <WelcomeToCre8ors />
+            <InviteForCreator />
+            <Networking />
+            <Cre8orsWay />
+            <Brands />
+            <OpenSoon />
+            <Footer />
+        </div>
```
Comment (sweetmantech) [inline components/LandingPage/LandingPage.tsx]: very clean component ✨

File: components/LandingPage/SectionContent.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import * as React from 'react'
+
+interface SectionContentProps {
+    children: React.ReactNode
+}
```
Comment (sweetmantech) [inline components/LandingPage/SectionContent.tsx:6]: nice interface definition 💯

File: components/LandingPage/SectionTitle.tsx [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import * as React from 'react'
```
Comment (sweetmantech) [inline components/LandingPage/SectionTitle.tsx]: nit

I don't think you need to import React unless you're explicitly using it

File: components/LandingPage/sections/Brands.tsx [hunk 5]
```diff
@@ -0,0 +1,86 @@
+import Image, { StaticImageData } from 'next/image'
+import * as React from 'react'
+
+import ExtendImage from '../../../assets/Landing/brands/extend.png'
+import LampImage from '../../../assets/Landing/brands/lamp.png'
+import ChainImage from '../../../assets/Landing/brands/chain.png'
+import NetworkingImage from '../../../assets/Landing/brands/networking.png'
```
Comment (sweetmantech) [inline components/LandingPage/sections/Brands.tsx]: in NextJS I believe it's best practice to store assets (like PNG / SVG) in the `public` directory.

Then, you can reference those assets with `/path-from-public.png`

https://nextjs.org/docs/app/building-your-application/optimizing/static-assets

File: components/_LandingPage/LandingPage.tsx [hunk 6]
```diff
@@ -0,0 +1,18 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```
Comment (sweetmantech) [inline components/_LandingPage/LandingPage.tsx]: why are you muting typescript checks here?

**Review by SweetmanTech — APPROVED** [review]
nit feedback

overall great first PR 🚢

File: components/LandingPage/SectionTitle.tsx [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+
+interface SectionTitleProps {
+    text: string
+}
+
+const SectionTitle = ({
```
Comment (crazzywizard) [inline components/LandingPage/SectionTitle.tsx]: FYI You can do `const SectionTitle:React.FC<SectionTitleProps> = ({SectionTitleProps}) => ...`

File: components/LandingPage/sections/Brands.tsx [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+        return (
+            <div className='font-quicksand lg:text-[1rem] md:text-[1rem] text-center w-[200px]'>
+                {children}
+            </div>
+        )
+    }
+
+    return (
+        <div className='lg:m-12 grid grid-cols-2 md:grid-cols-4 pt-[10rem]'>
+            {Items.map((item: ItemData, index: number) => (
```
Comment (crazzywizard) [inline components/LandingPage/sections/Brands.tsx]: NIT please don't use index for key, you can add key attribute to the list and use that.

File: components/LandingPage/SectionTitle.tsx [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+    text
+}: SectionTitleProps) => {
+    return (
+        <div className='lg:m-12 lg:text-6xl md:text-6xl font-eigerdals'>
```
Comment (crazzywizard) [inline components/LandingPage/SectionTitle.tsx]: in tailwind you can combine this to `lg:md:text-6xl`

File: components/CustomConnectWallet/CustomConnectWallet.tsx [hunk 10]
```diff
@@ -89,7 +93,7 @@ const CustomConnectWallet = () => (
                 <button
                   onClick={openAccountModal}
                   type="button"
-                  className="font-bold uppercase text-[17px] lg:md:text-sm font-quicksand"
+                  className="font-bold uppercase text-[17px] lg:md:text-sm font-quicksand text-[black] dark:text-[white]"
```
Comment (sweetmantech) [inline components/CustomConnectWallet/CustomConnectWallet.tsx:96]: 💯

File: components/DesktopMenu/DesktopMenu.tsx [hunk 11]
```diff
@@ -1,20 +1,44 @@
...
 import CustomConnectWallet from "../CustomConnectWallet"
+import { ToggleButton } from "../../shared/Button"
+import { useTheme } from "../../providers/ThemeProvider"
 
 const DesktopMenu = () => {
+  const { onChangeThemeConfig, themeMode } = useTheme()
+
   const [isMenuOpen, setIsMenuOpen] = useState(false)
   const toggleMenu = () => setIsMenuOpen(!isMenuOpen)
+  const [toggle, setToggle] = useState(false)
```
Comment (sweetmantech) [inline components/DesktopMenu/DesktopMenu.tsx]: can you change this to `isDarkMode` to make it distinct from `isMenuOpen`?

current name: `toggle` seems like a function call while this is a `boolean` right?

File: components/DesktopMenu/DesktopMenu.tsx [hunk 12]
```diff
@@ -1,20 +1,44 @@
...
+
+  const onToggle = () => {
+    setToggle(!toggle)
+    onChangeThemeConfig()
+  }
+
+  React.useEffect(() => {
+    if (themeMode === "light") setToggle(false)
+    if (themeMode === "dark") setToggle(true)
+  }, [themeMode])
```
Comment (sweetmantech) [inline components/DesktopMenu/DesktopMenu.tsx]:
```suggestion
  useEffect(() => {
  setToggle(themeMode !== "light")
  }, [themeMode])
```

Comment (sweetmantech) [reply components/DesktopMenu/DesktopMenu.tsx]: this accomplishes the same goal right?

File: components/LandingPage/sections/Brands.tsx [hunk 13]
```diff
@@ -0,0 +1,100 @@
...
+  image: StaticImageData
+  width: number
+  height: number
+  title: React.ReactNode
+  text: React.ReactNode
+  key: string
+}
+
+const Brands = () => {
+  const match490 = useMediaQuery("(max-width: 490px)")
```
Comment (sweetmantech) [inline components/LandingPage/sections/Brands.tsx]: why are we matching `490` exactly?

could we make this variable name more easy to understand its functionality?

`isMobile` maybe? What is the breakpoint you're checking here?

File: components/LandingPage/sections/Cre8orsWay.tsx [hunk 14]
```diff
@@ -0,0 +1,35 @@
+import Image from "next/image"
+import { useMediaQuery } from "usehooks-ts"
+import SectionTitle from "../SectionTitle"
+import SectionContent from "../SectionContent"
+
+import ProfileImage from "../../../public/assets/Landing/profile.png"
+
+const Cre8orsWay = () => {
+  const match490 = useMediaQuery("(max-width: 490px)")
```
Comment (sweetmantech) [inline components/LandingPage/sections/Cre8orsWay.tsx]: again, what is this `490` breakpoint? Can you rename this variable to be more clear?

File: hooks/useFadeIntersection.tsx [hunk 15]
```diff
@@ -0,0 +1,29 @@
+import * as React from 'react'
+
+interface Props {
+    ref: React.RefObject<HTMLDivElement>
+}
+
+const useFadeInstersection = ({
```
Comment (sweetmantech) [inline hooks/useFadeIntersection.tsx]:
```suggestion
const useFadeIntersection = ({
```

**Review by SweetmanTech — COMMENTED** [review]
overall beautiful changes  ✨ 

small feedback

ship it 🚢

File: components/DesktopMenu/DesktopMenu.tsx (hunk 12 above)
Comment (towardsnewtech) [reply components/DesktopMenu/DesktopMenu.tsx:24]: If users refresh pages, we should set theme mode from local storage variable.
So, I have integrated this part.

File: components/DesktopMenu/DesktopMenu.tsx [hunk 16]
```diff
@@ -1,20 +1,43 @@
...
+import DiscordIcon from "../DiscordIcon"
+import { ToggleButton } from "../../shared/Button"
+import { useTheme } from "../../providers/ThemeProvider"
 
 const DesktopMenu = () => {
+  const { onChangeThemeConfig, themeMode } = useTheme()
+
   const [isMenuOpen, setIsMenuOpen] = useState(false)
   const toggleMenu = () => setIsMenuOpen(!isMenuOpen)
+  const [isDarkMode, setIsDarkMode] = useState(false)
```
Comment (sweetmantech) [inline components/DesktopMenu/DesktopMenu.tsx:14]: 💯

**Review by SweetmanTech — APPROVED** [review]
looking good 🚢

File: components/DiscordIcon/DiscordIcon.tsx [hunk 17]
```diff
@@ -1,22 +1,27 @@
...
+import { useTheme } from "../../providers/ThemeProvider"
 
 const DiscordIcon = () => {
-  const isMobile = useMediaQuery("(max-width: 768px)")
+  const { themeMode } = useTheme()
 
   return (
     <a
-      href="https://discord.gg/ZpZBHCrqHQ"
+      href="https://discord.com/channels/1055217047908384849/1055218028012392519"
```
Comment (sweetmantech) [inline components/DiscordIcon/DiscordIcon.tsx]:
```suggestion
      href="https://discord.gg/ZpZBHCrqHQ"
```

please do not change the Discord link. This is important for new members of our NFT project to join our community.

File: components/Enter/index.tsx [hunk 18]
```diff
@@ -0,0 +1,50 @@
+import * as React from "react"
+
+import Image from "next/image"
+import { useRouter } from "next/router"
+import styles from "../../styles/Animation.module.css"
+import { Button } from "../../shared/Button"
+
+const EnterPage = () => {
```
Comment (sweetmantech) [inline components/Enter/index.tsx]: Can you please move this component to `components/Enter/EnterPage.tsx`?

We do not put component code in `index.tsx` files.

You can reference ANY component to see what I mean for an example of this architecture.

Does that make sense?

File: components/LandingPage/sections/Footer.tsx [hunk 19]
```diff
@@ -0,0 +1,96 @@
...
+              <input
+                className="border-[1px] border-[black] rounded-[3rem] font-quicksand px-3 py-1"
+                placeholder="Email Address"
+              />
+              <div className="absolute top-[10px] right-[0px] md:right-[10px]">
+                <button
+                  type="button"
+                  className="bg-[black] px-2 py-1 rounded-[2rem] text-[white] text-[0.5rem]"
+                >
+                  SUBMIT
```
Comment (sweetmantech) [inline components/LandingPage/sections/Footer.tsx]: Did you preserve the functionality we had for this button?

Looks like you got rid of it but wanted to check...

Comment (sweetmantech) [reply components/LandingPage/sections/Footer.tsx]: for clarity: Submit button in footer should preserve it's existing functionality.

File: components/LandingPage/sections/WelcomeToCre8ors.tsx [hunk 20]
```diff
@@ -0,0 +1,46 @@
...
+          <div className="m-6 lg:m-12 flex justify-center md:justify-start">
+            <Button id="welcome_reserve_btn" onClick={goToReserveList}>
+              Reserve list
+            </Button>
+            {/* <button
+              type="button"
+              className={`hover:scale-[1.25] transition duration-[500ms] px-4 py-2 font-bold font-quicksand text-white dark:text-[black] rounded bg-[black] dark:bg-[white] shadow-[0px_4px_4px_rgb(0,0,0,0.25)] dark:shadow-[0px_4px_4px_rgb(255,255,255,0.25)]`}
+            >
+              Reserve list
+            </button> */}
```
Comment (sweetmantech) [inline components/LandingPage/sections/WelcomeToCre8ors.tsx]: please remove commented code

principle: [YAGNI](https://www.techtarget.com/whatis/definition/You-arent-gonna-need-it#:~:text=YAGNI%20principle%20(%22You%20Aren',only%20be%20added%20when%20required)

File: components/LandingPage/sections/WelcomeToCre8ors.tsx [hunk 21]
```diff
@@ -0,0 +1,46 @@
...
+      <div className="flex flex-col justify-center">
+        <div>
+          <SectionTitle text="Welcome to the Next Generation of Creativity" />
+          <SectionContent>
+            Cre8ors is a next-gen media brand made for the metaverse; <br />
+            powered by our curated collective of web3 creators, IP co-creation protocols and
+            AI-enabled NFTs.
+          </SectionContent>
+          <div className="m-6 lg:m-12 flex justify-center md:justify-start">
+            <Button id="welcome_reserve_btn" onClick={goToReserveList}>
```
Comment (sweetmantech) [inline components/LandingPage/sections/WelcomeToCre8ors.tsx]: why are you using an `onClick` function with `window`?

why not use NextJS native options?
- `<Link />`
- `<a />`
- `router.push()`

using `window` feels like a code smell

Comment (sweetmantech) [reply components/LandingPage/sections/WelcomeToCre8ors.tsx]: I recommend wrapping your Button in a `<Link />` or `<a />` component

File: components/LandingPage/sections/WelcomeToCre8ors.tsx (hunk 20 above)
Comment (sweetmantech) [inline components/LandingPage/sections/WelcomeToCre8ors.tsx]: please remove commented code 🚮

File: components/Layout/BaseLayout.tsx [hunk 22]
```diff
@@ -0,0 +1,20 @@
+import React from "react"
+import { ILayout } from "./types"
+import SeoHead from "../SeoHead"
+import Header from "../Header"
+
+// const changeActivePage = () => {
+//   window.scrollTo({ top: 0, behavior: 'smooth' })
+// }
```
Comment (sweetmantech) [inline components/Layout/BaseLayout.tsx]: please remove unused code 🚮

File: components/Layout/ContainedLayout.tsx [hunk 23]
```diff
@@ -0,0 +1,24 @@
+import React from "react"
+import { ILayout } from "./types"
+import SeoHead from "../SeoHead"
+import Header from "../Header"
+import Footer from "../LandingPage/sections/Footer"
+
+// const changeActivePage = () => {
+//   window.scrollTo({ top: 0, behavior: 'smooth' })
+// }
```
Comment (sweetmantech) [inline components/Layout/ContainedLayout.tsx]: please remove commented code 🚮

File: components/Layout/index.tsx [hunk 24]
```diff
@@ -0,0 +1,21 @@
+import BaseLayout from "./BaseLayout"
+import ContainedLayout from "./ContainedLayout"
+import { ILayout } from "./types"
+
+const layoutContainers = {
+  base: BaseLayout,
+  contained: ContainedLayout,
+  // if needed - add more layout containers here
```
Comment (sweetmantech) [inline components/Layout/index.tsx]: please remove comments 🚮

File: styles/globals.css [hunk 25]
```diff
@@ -80,6 +80,31 @@ a {
...
+.fade_bg {
+  opacity: 0.4;
+  /* transform: translate(0, 20px); */
+  transition: all 2s;
+}
+
+.fade_bg.show {
+  opacity: 1;
+  /* transform: translate(0, 0); */
+}
```
Comment (sweetmantech) [inline styles/globals.css:106]: it looks like you're doing all animations in CSS right?

that's fine, just want to make you aware we also have Framer Motion for JS-based animations.

Your call 🌈 

docs: https://www.framer.com/motion/

**Review by SweetmanTech — COMMENTED** [review]
Discord link should not be changed and Submit button in Footer needs to be functional.

**Review by SweetmanTech — APPROVED** [review]
ship it 🚢

**Pull request merged**